zgw_consumers/models/certificates.py

```diff
@@ -0,0 +1,99 @@
+from datetime import date, datetime
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from OpenSSL import SSL, crypto
+from privates.fields import PrivateMediaFileField
+
+from ..constants import CertificateTypes
+from ..mixins import DeleteFileFieldFilesMixin
+from ..utils import pretty_print_certificate_components
+
+
+def ssl_upload_to(instance, filename):
+    return "ssl_certs_keys/{d}".format(d=date.today().strftime("%Y/%m/%d"))
```

Comment:
`filename` is missing from this path - you should `posixpath.join(<dirpath>, filename)` this.

Reply:
I was looking at the documentation of the `FileField` and realised that the filename is not included in the `upload_to` https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.FileField.upload_to :thinking: 

So maybe the `upload_to` in the model should just be the string `ssl_certs_keys/%Y/%m/%d` instead of having a function returning a string?